src/main/java/org/folio/search/service/browse/CallNumberBrowseService.java

```diff
@@ -107,6 +107,14 @@ private BrowseResult<CallNumberBrowseItem> additionalPrecedingRequests(BrowseReq
     return precedingResult;
   }
 
+  private String getPrevBrowsingAroundValue(List<CallNumberBrowseItem> precedingResult,
+                                            List<CallNumberBrowseItem> succeedingResult, BrowseContext ctx) {
+    if (isEmpty(precedingResult)) {
+      return getPrevBrowsingValue(succeedingResult, ctx, true);
```

Comment:
I don't think that it has the sense to get the 'previous' value from 'succeedingResult'. With this implementation we always will have some value in 'previous' field, even if there is no preceding page exists for the request. And a test `browseByCallNumber_browsingVeryFirstCallNumberWithNoException` proves this concern.